cdap-spark-core-base/src/main/java/io/cdap/cdap/app/runtime/spark/SparkProgramRuntimeProvider.java

```diff
@@ -90,6 +90,10 @@ public ProgramRunner createProgramRunner(ProgramType type, Mode mode, Injector i
     Preconditions.checkArgument(type == ProgramType.SPARK, "Unsupported program type %s. Only %s is supported",
                                 type, ProgramType.SPARK);
 
+    boolean rewriteCheckpointTempFileName = injector.getInstance(CConfiguration.class)
```

Comment:
Better get the `CConfiguration` instance once as a local variable and use it throughout this method.

Reply:
Done!